Release.toml

```diff
@@ -25,6 +25,6 @@ version = "1.0.5"
     "migrate_v1.0.6_add-static-pods.lz4",
     "migrate_v1.0.6_kubelet-standalone-tls-settings.lz4",
     "migrate_v1.0.6_kubelet-standalone-tls-services.lz4",
-    "migrate_v1.0.6_control-container-v0-4-2.lz4"
+    "migrate_v1.0.6_control-container-v0-4-2.lz4",
```

Comment:
I believe this is pointing to the need for a rebase - we should only see shibaken in the diff for the migration list.